### ISSUE TITLE
EXCLUDE variable in LINT test does not ignore empty space in comma separeted list

### ISSUE THREAD

**Issue by tafaRU** [OCA/maintainer-quality-tools#668]
EXCLUDE variable has different beaviour depending on whether it is used:

* TEST --> test_server.py https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/travis/test_server.py#L108
* LINT:
  * test_flake8.py https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/travis/test_flake8#L14
  * run_pylint.py https://github.com/OCA/maintainer-quality-tools/blob/67d3880b085a99d820e5a65cc7354dae95d1b4b0/travis/run_pylint.py#L278

`strip()` also should be applied to LINT CASE in order to allow to define the variable in the same way for both tests.

**Comment by pedrobaeza** [comment]
Well, that hasn't been a problem since now if you don't put spaces between module names in EXCLUDE section, but please do a PR if you want to homogenize the behavior.

**Comment by tafaRU** [comment]
> Well, that hasn't been a problem since now if you don't put spaces between module names in EXCLUDE section

Is it documented? I had to take some digging before I found out that!

> please do a PR if you want to homogenize the behavior.

Sure!

**Comment by pedrobaeza** [comment]
No, not documented, but first you try and see the result, you can deduce it :wink: In fact it has been the other way: there was no support for spaces, and someone added it to test part but not the others.

**Comment by tafaRU** [comment]
> and someone added it to test part but not the others

This is strange! Anyway, I'll do a PR to homogenize the behavior.
